libs/ui-components/src/components/DetailsPage/DetailsPageCard.tsx

```diff
@@ -1,10 +1,17 @@
-import { Card, CardBody, CardBodyProps, CardProps } from '@patternfly/react-core';
+import { Card, CardBody, CardBodyProps, CardProps, CardTitle, Divider } from '@patternfly/react-core';
 import * as React from 'react';
 
 import './DetailsPageCard.css';
 
 const DetailsPageCard: React.FC<CardProps> = (props) => <Card className="fctl-details-page__card" {...props} />;
 
+export const DetailsPageCardTitle = ({ title }: { title: string }) => (
+  <CardTitle className="fctl-details-page__card-title">
+    {title}
+    <Divider component="div" />
+  </CardTitle>
+);
```

Comment:
instead of adding a custom css, we could use `Stack`
```suggestion
  <CardTitle>
    <Stack hasGutter>
      <StackItem>
        {title}
      </StackItem>
      <StackItem>
        <Divider component="div" />
      </StackItem>
    </Stack>
  </CardTitle>
);
```

Reply:
I added the custom CSS because the divider line width seemed to not work well without that, but now I realize that it depends a bit on the viewport width, so it's not really needed.

But for reusability, wouldn't it be preferable to have our own component? The Overview page seems to have a very similar pattern, so I'd rather not DRY (actually, I'd argue that the pattern in Overview and Device details should be the same).

Is is OK if I remove the custom CSS and keep the component?

Reply:
I removed the custom component with the divider and its CSS based on the discussion on Slack.